src/components/language/language.js

```diff
@@ -160,7 +160,8 @@ class LanguageControl extends BaseComponent {
    * @returns {React$Element} - component
    * @memberof LanguageControl
    */
-  renderAll(audioOptions: Array<Object>, textOptions: Array<Object>): React$Element<any> {
+  renderAll(audioOptions: Array<Object>, textOptions: Array<Object>): React$Element<any> | void {
+    if (audioOptions.length <=1 && textOptions.length <= 1) return undefined;
```

Comment:
the `render` method already has condition of the length. edit it instead of adding a new condition.

Reply:
what if audio has 1 the text has 2. do we want to render all?

Reply:
beautify :)

Reply:
same as this: https://github.com/kaltura/playkit-js-ui/pull/253/files#r201715680